ds3-autogen-go/src/main/kotlin/com/spectralogic/ds3autogen/go/generators/parser/TypeParserGeneratorUtil.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2017 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3autogen.go.generators.parser
+
+interface TypeParserGeneratorUtil {
```

Comment:
This interface will be used to specify which functions have different implementations when I go to special case the parser generators. Its currently just a placeholder.